src/BaldinofRoadRunnerBundle.php

```diff
@@ -17,5 +18,8 @@ public function build(ContainerBuilder $container): void
 
         $container->addCompilerPass(new RemoveConfigureVarDumperListenerPass());
         $container->addCompilerPass(new MiddlewareCompilerPass());
+        if (interface_exists(ServiceInterface::class)) {
```

Comment:
shouldn't this interface be imported as `Spiral\RoadRunner\GRPC\ServiceInterface` ?

maybe it should also be noted in the docs that only `Spiral\RoadRunner\GRPC\ServiceInterface` is supported, not the deprecated `Spiral\GRPC\ServiceInterface`.
I had this problem because I stubs built by the outdated https://github.com/spiral/php-grpc.git which is for roadrunner v1 :)

Reply:
PP: would add `use function interface_exists;`

Reply:
Yes, I missed the import! For the `use function` I let php-cs-fixer decide if a `\` should be added to native function thanks to this config: https://github.com/Baldinof/roadrunner-bundle/blob/2.x/.php-cs-fixer.dist.php#L16